app-misc/cal/cal-4.1.recipe

```diff
@@ -0,0 +1,41 @@
+SUMMARY="Display a monthly or yearly calendar, with optional appointments"
+DESCRIPTION="By default, cal will display a calendar for the current month \
+with the current day marked. By specifing certain arguments, cal will \
+display a calendar for a whole year or a specified month and year."
+HOMEPAGE="http://unicorn.us.com/cal.html"
+COPYRIGHT="1993-2014, Alex Matulich, by Unicorn Research Corporation."
```

Comment:
I think we prefer to drop the first comma. And maybe we should drop the `by ` too (but I'm not sure about that.)
```diff
-COPYRIGHT="1993-2014, Alex Matulich, by Unicorn Research Corporation."
+COPYRIGHT="1993-2014 Alex Matulich, Unicorn Research Corporation."
```
That said, I'm ok with what we have right now :)

Reply:
@your service :)

Reply:
@alaviss, I think the PR is ready for a merge.
(All your comments have been addressed, and I have also checked that it builds on all arches :)